README.md

```diff
@@ -117,20 +151,41 @@ The former is a simple example using only Asp.NET components, and the latter is
 
 ```javascript
 {
-    "profile": "default",
+    "profile": "default", // AWS connexion profile
     "configuration": "Release",
     "framework": "net6.0",
     "function-handler": "Estranged.Lfs.Hosting.Lambda::Estranged.Lfs.Hosting.Lambda.LambdaEntryPoint::FunctionHandlerAsync",
     "function-memory-size": 256,
     "function-timeout": 30,
     "function-runtime": "dotnet6",
-    "region": "<aws region>",
-    "s3-bucket": "<s3 bucket to upload the lambda to>",
+    "region": "<aws region>", // AWS public region
+    "s3-bucket": "<s3 bucket to upload the lambda to>", // S3 bucket needed to upload the modele/output of the stack, must be outside of the stack (shared between all stacks)
     "s3-prefix": "<path in s3 to upload the lambda to>",
-    "function-name": "<lambda name to deploy or update>",
+    "function-name": "<lambda name to deploy or update>", // lambda name must be same as stack name
     // Set other variables required by the Lambda function
-    "environment-variables": "LFS_BUCKET=<lfs s3 bucket>;<key>=<value>"
+    "environment-variables": "LFS_BUCKET=<lfs s3 bucket>;LFS_USERNAME=<AWS_STACK_ParameterUsername>;LFS_PASSWORD=<AWS_STACK_ParameterPassword>;S3_ACCESS_KEY=<S3 AccessKey>;S3_ACCESS_SECRET=<S3 AccessSecret>;S3_REGION=<Custom S3 Region>;S3_SERVICE_URL=<Custom S3 EndPoint>;<key>=<value>", // can be found and changed in Lambda configuration UI"
 }
 ```
+4. Run `dotnet lambda deploy-serverless` to deploy the stack
+5. Run `dotnet lambda deploy-function` to deploy the code of the lambda function
```

Comment:
These should not be used together, only one or the other - the stack should deploy the Lambda function code

Reply:
The 5. is used when the stack is already deployed and we want to redeploy only lambda code. Let me precise it.